src/backend/joanie/payment/backends/dummy.py

```diff
@@ -154,16 +154,16 @@ def _get_payment_data(
     def create_payment(
         self,
         order,
+        installment,
         billing_address=None,
-        installment=None,
     ):
```

Comment:
in the `_get_payment_data` you should remove this check : 

```
        if installment:
            payment_info["metadata"]["installment_id"] = installment["id"]
```

and directly do : 

```
         payment_info["metadata"]["installment_id"] = installment["id"]
```